src/app/view-config-upgraders.js

```diff
@@ -218,8 +219,18 @@ export function upgradeFrom1_0_1(config) {
     return newComponent;
   });
 
+  // Enforce bitmask or raster as spatial raster layer type, defaulting
+  // to raster layer if it is not one of bitmask or raster from the old config.
+
+  const newConfig = cloneDeep(config);
+  Object.keys(newConfig.coordinationSpace.spatialRasterLayers).forEach((key) => {
```

Comment:
I think there should be a check for whether the `coordinationSpace` object and the `coordinationSpace.spatialRasterLayers` array exist here before accessing the sub-values

Reply:
Oh, good call.  There is definitely no way they will definitely be defined.  I only tried this out on the problematic portal config.